### PR TITLE
Update release notes links and generating script

### LINKED ISSUE #858: Update release notes link in vsix manifest and nuget spec
The link points to old release notes, which is not valid after the website redesign.

See https://github.com/SonarSource/sonar-csharp/pull/856#discussion_r145893728

SonarC# Version: 6.5

### PR DESCRIPTION
**Pull request by Evangelink**
Fix #858

### CODE REVIEW

File: sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.VisualBasic.nuspec [hunk 1]
```diff
@@ -12,7 +12,8 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <summary>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio and/or the SonarQube platform.</summary>
     <description>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio (http://vs.sonarlint.org) and/or the SonarQube platform (http://www.sonarqube.org).</description>
-    <releaseNotes>http://www.sonarlint.org/visualstudio/rules/index.html#version=6.6.0.0</releaseNotes>
+    <!-- The link to the release notes needs to be updated manually (the version part) -->
+    <releaseNotes>https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10112&version=13300</releaseNotes>
```
Comment (duncanp-sonar) [inline sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.VisualBasic.nuspec]: Doesn't this ampersand need to be escaped too?

**Comment by Evangelink** [comment]
@duncanp-sonar was escaped while you were reviewing :)

**Comment by duncanp-sonar** [comment]
👍 :-)

File: sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.VisualBasic.nuspec [hunk 2]
```diff
@@ -12,7 +12,8 @@
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <summary>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio and/or the SonarQube platform.</summary>
     <description>Analyzers which spot bugs and code smells in your code. This package is best used together with SonarLint for Visual Studio (http://vs.sonarlint.org) and/or the SonarQube platform (http://www.sonarqube.org).</description>
-    <releaseNotes>http://www.sonarlint.org/visualstudio/rules/index.html#version=6.6.0.0</releaseNotes>
+    <!-- The link to the release notes needs to be updated manually (the version part) -->
+    <releaseNotes>https://jira.sonarsource.com/jira/secure/ReleaseNote.jspa?projectId=10112&amp;version=13300</releaseNotes>
```
Comment (michalb-sonar) [inline sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.VisualBasic.nuspec]: Make sure this manual step is described in release process (if not there already).

**Pull request merged**